### PR TITLE
[MIG] Migration partner_contact_birthdate partner_contact_gender part…

### PR DESCRIPTION
**Pull request by cvinh**
…ner_contact_personal_information_page to v10

### CODE REVIEW

**Comment by Garamotte** [comment]
As said in https://github.com/OCA/partner-contact/pull/307, you should add the post init hook to call `_birthdate_install` in the partner_contact_birthdate module, and remove the yaml file.

Also, I think you should split the migration commit, one per module.

Otherwise, :+1: (code review + functional tests)

File: partner_contact_birthdate/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 {
     "name": "Contact's birthdate",
-    "version": "9.0.1.0.0",
+    "version": "10.0.1.0.0",
     "author": "Jairo Llopis,"
```
Comment (rafaelbn) [inline partner_contact_birthdate/__manifest__.py]: Please add Tecnativa, we join all together in a new team.

File: partner_contact_gender/__manifest__.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 {
     "name": "Contact gender",
     "summary": "Add gender field to contacts",
-    "version": "9.0.1.1.0",
+    "version": "10.0.1.1.0",
     "category": "Customer Relationship Management",
     "website": "https://odoo-community.org/",
```
Comment (rafaelbn) [inline partner_contact_gender/__manifest__.py]: Please change to https://www.tecnativa.com as we are in the new team

File: partner_contact_gender/__manifest__.py [hunk 3]
```diff
@@ -5,7 +5,7 @@
...
     "author": "Grupo ESOC, Odoo Community Association (OCA)",
```
Comment (rafaelbn) [inline partner_contact_gender/__manifest__.py]: Please add Tecnativa, we join all together in a new team

**Review by rafaelbn — CHANGES_REQUESTED** [review]
Hi @invitu thanks for migration. But please make a single PR per module. It helps a lot 😄

**Comment by pedrobaeza** [comment]
@rafaelbn, in this case, we can accept all the modules together are they are simple and bound.

**Comment by rafaelbn** [comment]
Sorry I put `need fixing` label until attended https://github.com/OCA/partner-contact/pull/307#issuecomment-253670296 thanks!

**Comment by rafaelbn** [comment]
OK @pedrobaeza just minor comments and https://github.com/OCA/partner-contact/pull/307#issuecomment-253670296

**Comment by cvinh** [comment]
I did a PR for those 3 modules because they are small and depend on partner_contact_personal_information_page

requested fixes are commited

**Review by lasley — APPROVED** [review]
LGTM code & test

**Review by rafaelbn — APPROVED** [review]

**Review by rafaelbn — APPROVED** [review]
@pedrobaeza ready to merge, tested in runbot

File: partner_contact_birthdate/__manifest__.py [hunk 4]
```diff
@@ -4,16 +4,19 @@
...
+              "Tecnativa,"
               "Odoo Community Association (OCA)",
```
Comment (rafaelbn) [inline partner_contact_birthdate/__manifest__.py:9]: Shouldn't it be "Jairo Llopis, Tecnativa, Odoo Community Association (OCA)" ?

**Pull request merged**